webapp/channels/src/i18n/en.json

```diff
@@ -1922,6 +1922,7 @@
   "admin.plugin.backToPlugins": "Go back to the Plugins",
   "admin.plugin.choose": "Choose File",
   "admin.plugin.cluster_instance": "Cluster Instance",
+  "admin.plugin.customSection.pluginDisabledWarning": "In order to view this section, enable the plugin and click Save.",
   "admin.plugin.customSetting.pluginDisabledWarning": "In order to view this setting, enable the plugin and click Save.",
```

Comment:
Is this line doubled?

Reply:
Nope, it's for the section rather than the setting.

Reply:
Ah, missed the difference, thanks.